File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.py
class Solution(object):
    def titleToNumber(self, columnTitle):
        """
        :type columnTitle: str
        :rtype: int
        """
        colNum=0
        for char in columnTitle:
            
            value=ord(char)-ord('A')+1
            colNum=colNum*26+value
        return colNum
        

File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.java
class Solution {
    public int titleToNumber(String columnTitle) {
        char[] ch=columnTitle.toCharArray();
        int colNum=0;
        for(int i=0;i<ch.length;i++){
            char c=ch[i];
            int value=c-'A'+1;
            colNum=colNum*26+value;
        }
        return colNum;
        
    }
}